### DISCUSSION TITLE
В таблице циклов скрыть таблицы по условию

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#234] [Есть идея, а что если...]
В таблице циклов скрыть вкладки таблиц по условию

Насколько я понял по видео урокам это можно сделать, но в уроках это показано только доступам по ролям. А как можно скрыть вкладки таблиц по условию? То есть проверить если в первой таблице стоит галочка в одном из полей, и в зависимости от значения скрыть вкладки в цикле.

**Comment by ttm-alexeypolunin** [comment]
Не делали. Запишу в PRO
